src/sentry/relay/config/__init__.py

```diff
@@ -170,9 +171,26 @@ def get_filter_settings(project: Project) -> Mapping[str, Any]:
     if csp_disallowed_sources:
         filter_settings["csp"] = {"disallowedSources": csp_disallowed_sources}
 
+    try:
+        generic_filters = _get_generic_project_filters()
+    except Exception:
+        logger.exception(
+            "Exception while building Relay project config: error building generic filters"
+        )
```

Comment:
Is this better than `sentry_sdk.capture_exception()`?

Reply:
Not sure if it's better, but these exceptions should also reach S4S.